crates/engine/invalid-block-hooks/src/witness.rs

```diff
@@ -185,6 +193,36 @@ where
             }
         }
 
+        if let Some(healthy_node_client) = &self.healthy_node_client {
+            // Compare the witness against the healthy node.
+            let healthy_node_witness = futures::executor::block_on(async move {
+                DebugApiClient::debug_execution_witness(
+                    healthy_node_client,
+                    block.number.into(),
+                    true,
+                )
+                .await
+            })?;
+
+            // Write the healthy node witness to the output directory.
+            File::options()
+                .write(true)
+                .create_new(true)
+                .open(self.output_directory.join(format!(
+                    "{}_{}.healthy_witness.json",
+                    block.number,
+                    block.hash()
+                )))?
+                .write_all(serde_json::to_string(&healthy_node_witness)?.as_bytes())?;
```

Comment:
could we just `fs::write` here

Reply:
I wanted an explicit error when trying to overwrite the already existing output

Reply:
ah, I see, then maybe we can just `File::create_new` because that does:
```
File::options().read(true).write(true).create_new(true).open(...)
```